secrethub/resource_access_rule.go

```diff
@@ -0,0 +1,84 @@
+package secrethub
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/secrethub/secrethub-go/internals/api"
+)
+
+func resourceAccessRule() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAccessRuleSet,
+		Read:   resourceAccessRuleRead,
+		Update: resourceAccessRuleSet,
+		Delete: resourceAccessRuleDelete,
+		Schema: map[string]*schema.Schema{
+			"dir_path": {
```

Comment:
Maybe just `path` here, instead of `dir_path`?

Reply:
Depends, does it also support secrets?

Reply:
Nope, just directories

Reply:
And what about just `dir`? Or does that make it unclear whether we want the path or the ID? We might want to accept the ID as well later, but we currently do not yet do so.

Reply:
Let's stick with `dir_path`, I'd say. Not sure a dir ID is something we'd like to expose to the users.